packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -263,6 +270,26 @@ function run(task) {
   }
 }
 
+function validateFile(actualCode, expectedLoc, expectedCode) {
+  try {
+    expect(actualCode).toEqualFile({
+      filename: expectedLoc,
+      code: expectedCode,
+    });
+  } catch (e) {
+    if (!process.env.OVERWRITE) throw e;
+
+    console.log(`Updated test file: ${expectedLoc}`);
```

Comment:
If we don't restore the `console.log` mock, would this line go to the `actualLogs`?

Reply:
That's right

Reply:
I don't think this is expected. `Updated test file: foo.js` is used to remind the developer and therefore should always print to stdout, regardless of `validateLogs`.

Reply:
Yeah, I meant: "You are right, I will fix it" :sweat_smile: